### ISSUE TITLE
nbt blocade

### ISSUE THREAD

**Issue by wheresrhys** [Financial-Times/next-build-tools#273]
To cancel all current builds

Useful for preventing a broken component from landing in prod

**Comment by matthew-andrews** [comment]
how about:-
```
nbt cancel-builds --provider jenkins
nbt cancel-builds --provider circle
nbt cancel-builds --provider circle --serves user-page
```

- **`provider`**, which ci provider, default both
- **`serves`**, the kind of app, default all

**Comment by wheresrhys** [comment]
As long as `--provider` isn't required - it'd put the onus on the dev to find out which ci handles an app, which kind of defeats the point of this, which is to be an emergency, no faff off-switch

**Comment by matthew-andrews** [comment]
yep, sorry that wasn't clear. both of those were optional

**Comment by wheresrhys** [comment]
sounds good then

**Comment by wheresrhys** [comment]
How about also a `--since 30m` option which will also rollback heroku apps that deployed within the given timeframe?

**Comment by matthew-andrews** [comment]
umm i think rollback ought to be separate !

**Comment by wheresrhys** [comment]
yep, fair enough, but would be valuable to have, even though heroku's admin screens are perfect

**Comment by matthew-andrews** [comment]
yes. quite. perfect.
